fragmentCodeAndTransport.adoc

```diff
@@ -207,20 +212,26 @@ byte 0, 00 is byte 1 and and the final value 20 is byte 9.
 
 ==== Siemens transport
 
-The packet format is given in
-<<fig:packet-format>>. So this means the packet
-will be packed as follows:
+The packet is encapsulated according to the https://github.com/riscv-non-isa/e-trace-encap/releases/latest/Unformatted Trace & Diagnostic Data Packet Encapsulation for RISC-V Specification], with the following attributes: 
 
 * Header - 1 byte
-* Index - N bits. As an example use 6 bits and the value of 0xA
-* Optional Siemens timestamp - 2 bytes. This example has no timestamp
-* A type field for the packet of 2 bits '01' meaning instruction trace
-* Payload - [0xBD 0xAA 0xAA 0x68 0x00 0x00 0x20]
+* SrcID - N bits. As an example use 6 bits and the value of A.
+* This example has no timestamp
+* A 2-bit type field with ’10’ meaning instruction trace
+* trace_payload - [0xBD 0xAA 0xAA 0x68 0x00 0x00 0x20]
 
-`[0x7][0x29][0xBD 0xAA 0xAA 0x68 0x00 0x00 0x20]`
+`[0x8][0x8A][0xBD 0xAA 0xAA 0x68 0x00 0x00 0x20]`
 
 ==== ATB transport
 
+The packet is encapsulated according to the https://github.com/riscv-non-isa/e-trace-encap/releases/latest/[Unformatted Trace & Diagnostic Data Packet Encapsulation for RISC-V Specification], with the following attributes:
+
+* Header - 1 byte
+* SrcID - 0 bits as the SrcID is carried via the ATID field.
+* This example has no timestamp
+* No type field (encoder has no data trace support)
```

Comment:
what happens when there is a need to carry type but no srcID - does it get 0 padded to a byte?

Reply:
The type field is defined as some number of LSBs in the payload, fixed per Src.  So if this source included a type field in its packets then it would be the appropriate number of LSBs of the payload, and the remaining payload would be left-shifted by that number of bits.  If necessary, the packet length may need to be increased (if the left-shift moved the most significant bit across a byte boundary).  

I chose to leave it out because it makes the example easier to read.  The hex values on the ATDATA bus would not be the same as the trace packet payload bytes.  For the same reason, I used a 6-bit SrcID and 2-bit type in the Siemens example so that the combined SrcID + type was a byte, leaving the trace packet bytes invariant.